src/Google/Ads/GoogleAds/Lib/V1/LogMessageFormatter.php

```diff
@@ -97,7 +117,7 @@ public function formatDetail(
 
         $response = $this->pluck('response', $responseData);
         $status = $this->pluck('status', $responseData);
-        $unaryCall = $this->pluck('unaryCall', $responseData);
+        $call = $this->pluck('call', $responseData);
```

Comment:
If you want to make this change you will also need to update [here](https://github.com/googleads/google-ads-php/blob/master/src/Google/Ads/GoogleAds/Lib/V1/GoogleAdsLoggingUnaryCall.php#L57) and [here](https://github.com/googleads/google-ads-php/blob/master/src/Google/Ads/GoogleAds/Lib/V1/GoogleAdsLoggingUnaryCall.php#L61).

Reply:
Done.